app/connectors/Acc44Connector.scala

```diff
@@ -0,0 +1,166 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package connectors
+
+import config.AppConfig
+import config.MetaConfig.Platform.MDTP
+import models.requests.{CashAccountTransactionSearchRequest, CashAccountTransactionSearchRequestContainer,
```

Comment:
give how long this is I'd either break it up so like

import abc.123.{
  itemOne,
  itemTwo}
  
  
 or just do ._ imo

Reply:
Adam, its okay for imports like this as its within 120 characters and on two lines. Cannot import all the other models that are not required as this adds the unnecessary overhead for the class and memory.